### DISCUSSION TITLE
How to edit and add to documentation?

### DISCUSSION THREAD

**Discussion by brjhaverkamp** [Koenkk/zigbee2mqtt#21913] [General]
Hello all, 

Maybe a weird question.
I am playing with a NAS-AB02B2 Siren Alarm in Z2M. 
I thought I could improve the documentation a bit with some settings and the pairing procedure. 
But  in the NAS-AB02B2.md file (and all other devices doc .md files
There is the explicite warning:

So am I missing something? Where do I add the text to the file. Should I disregard this section and just add to the .md file? 
Or should I add the text somewhere else? If so where?
And if I add text, how can I test the layout of the result?

I found the Contributing.md file, but that one does not mention documentation either (yet)

Kind regards,

Bert

**Comment by Koenkk** [comment]
Click on the "help to make..." at the bottom of the page https://www.zigbee2mqtt.io/devices/NAS-AB02B2.html

**Comment by brjhaverkamp** [comment]
Hi Koen,

Thanks for your answer.  And I see now that the critical part of my mail wasn't copied in correctly. Note to self: When copying in a comment line, it doesn't show if you don't make it verbatim :-) My Appologies, without that part the issue looks weird and very dumb.

Let me try again. I found the link indeed. That's how I got to the github repository There it states that I should fork the  project. OK, also easily done. But then in that file
https://github.com/Koenkk/zigbee2mqtt.io/edit/master/docs/devices/NAS-AB02B2.md
it clearly states:
```
<!-- !!!! -->
<!-- ATTENTION: This file is auto-generated through docgen! -->
<!-- You can only edit the "Notes"-Section between the two comment lines "Notes BEGIN" and "Notes END". -->
<!-- Do not use h1 or h2 heading within "## Notes"-Section. -->
<!-- !!!! -->
```
So should I disregard this comment and edit the text underneath or is there an other place where I should edit the text
Regards,

Bert

**Reply by Koenkk** [reply]
Everything below the notes section is generated from [here](https://github.com/Koenkk/zigbee-herdsman-converters/blob/31fa55f8ab399813697dc754865aedc1cdc72c73/src/devices/neo.ts#L46). 

But if you want to:

> I thought I could improve the documentation a bit with some settings and the pairing procedure.

Then just add that between:

```
<!-- Notes BEGIN: You can edit here. Add "## Notes" headline if not already present. -->


<!-- Notes END: Do not edit below this line -->
```

**Comment by brjhaverkamp** [comment]
Ah, thanks. now I get the logic.  I'll add some info there  and generate a pull request.
